.github/workflows/app_build.yml

```diff
@@ -72,27 +88,32 @@ jobs:
           SDKROOT: /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX.sdk
           QT_OS_NAME: mac
           QT_ARCH: clang_64
-        - os: windows-2019
+        - os: macos-arm64
           allow_failure: false
-          PLATFORM_NAME: Windows
+          SELF_HOSTED: true
+          PLATFORM_NAME: Darwin
           CPACK_BINARY_DEB: OFF
           CPACK_BINARY_IFW: ON
-          CPACK_BINARY_TGZ: OFF
-          CPACK_BINARY_ZIP: ON
-          BINARY_EXT: exe
-          COMPRESSED_EXT: zip
-          BINARY_PKG_PATH: _CPack_Packages/win64/IFW
-          COMPRESSED_PKG_PATH: _CPack_Packages/win64/ZIP
-          QT_OS_NAME: windows
-          QT_ARCH: win64_msvc2019_64
+          CPACK_BINARY_TGZ: ON
+          CPACK_BINARY_ZIP: OFF
+          BINARY_EXT: dmg
+          COMPRESSED_EXT: tar.gz
+          BINARY_PKG_PATH: _CPack_Packages/Darwin/IFW
+          COMPRESSED_PKG_PATH: _CPack_Packages/Darwin/TGZ
+          MACOSX_DEPLOYMENT_TARGET: 10.14
```

Comment:
Hmm, I see 10.15 in both EnergyPlus and OpenStudio.  Agree that we could be less accommodating if we need, I'd say we update to 10.15 unless there are problems in 10.15 and then go to 11.

Reply:
I build the arm64 packages myself, so that's local to my machine. see https://github.com/NREL/EnergyPlus/releases/tag/v22.2.0 for eg, you'll see the (only) arm64 package has 12.1 in the name.

Couple of datapoints:

* The Mac Mini (M1, 2020), which was as far as I know the first commercial M1 mac, was on Big Sur, so 11.x. At that point in time, I used Irvine's mac mini to build E+ package, and set the deployment target to 11.2
* My MBP M1 shipped with Monterrey (12.x) I think, or I shortly upgraded after
* I reckon people with M1 machines are more likely to be on the bleeding edge anyways.

So I figured I could just bump to 12.1, to match the macos-12 github actions runner.

Reply:
Yeah I get linker warnings that 12.1 is newer than the target 11.0 when I link with the OpenStudio build, did you build the OpenStudio packages for M1 with 12.1 too?  

`warning: dylib (../../OpenStudio-3.5.0/OpenStudio-3.5.0+7b14ce1588-Darwin-arm64/lib/libopenstudiolib.dylib) was built for newer macOS version (12.1) than being linked (11.0)`

If so, I guess there is no reason to try to be more backward compatible than OpenStudio is. Looks like the conan stuff is being built for 11.2.